lib/src/compile.ts

```diff
@@ -42,13 +45,15 @@ export async function compileString(options: {
   source: string;
   sourceMap?: (sourceMap: RawSourceMap) => void;
   url?: URL | string;
+  indentedSyntax?: boolean;
```

Comment:
We will eventually need to support all of the syntaxes specced here: https://github.com/sass/sass/blob/main/spec/js-api/options.d.ts.

So instead of `indentedSyntax?: boolean`, this should be `syntax?: Syntax` where `Syntax` is a new type: `export type Syntax = 'scss' | 'indented' | 'css';`.

Please also update the tests in `compile.test.ts`.